scripts/sns/sns.coffee

```diff
@@ -86,7 +86,12 @@ class SNS
     @robot.emit 'sns:notification', message
     @robot.emit 'sns:notification:' + message.topic, message
 
+  channelID: (name) ->
+    temp = name.toLowerCase().replace(/\s/g, "_")
+    "#{process.env.HUBOT_HIPCHAT_JID.split("_")[0]}_#{temp}@conf.hipchat.com"
```

Comment:
[MEMO] This feature is for hipchat users.
If HUBOT_HIPCHAT_JID is not set, an error happens.

```
[Tue Aug 09 2016 12:18:13 GMT+0000 (UTC)] ERROR TypeError: Cannot call method 'split' of undefined
  at SNS.channelID (/tmp/hubot-aws/scripts/sns/sns.coffee:91:36, <js>:92:50)
  at EventEmitter.<anonymous> (/tmp/hubot-aws/scripts/sns/sns.coffee:96:5, <js>:103:36)
  at EventEmitter.emit (events.js:95:17)
  at Robot.emit (/tmp/mtburn_hubot_dev/node_modules/hubot/src/robot.coffee:583:18, <js>:459:41)
  at SNS.notify (/tmp/hubot-aws/scripts/sns/sns.coffee:86:5, <js>:85:18)
  at SNS.process (/tmp/hubot-aws/scripts/sns/sns.coffee:62:7, <js>:55:21)
  at /tmp/hubot-aws/scripts/sns/sns.coffee:46:11, <js>:35:26
  at /tmp/hubot-aws/scripts/sns/support/sns_message_verify.coffee:57:12, <js>:77:14
  at IncomingMessage.<anonymous> (/tmp/hubot-aws/scripts/sns/support/sns_message_verify.coffee:21:14, <js>:29:16)
  at IncomingMessage.EventEmitter.emit (events.js:117:20)
  at _stream_readable.js:920:16
  at process._tickCallback (node.js:415:13)
```